app/views/stories/tagged_articles/_meta.html.erb

```diff
@@ -1,4 +1,4 @@
-<% display_tag_name = @tag.name.capitalize %>
+<% display_tag_name = @tag.pretty_name ? @tag.pretty_name : @tag.name.capitalize %>
```

Comment:
I think this can be shortened to `@tag.pretty_name || @tag.name.capitalize`, but maybe this decision about what to display as the tag name also belongs to the tag, and not the view template?